tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -975,8 +976,8 @@ static PyObject * THPVariable_to(PyObject* self, PyObject* args, PyObject* kwarg
   auto copy = std::get<3>(parsed);
   auto opt_memory_format = std::get<4>(parsed);
   auto& self_ = THPVariable_Unpack(self);
-  if (device && device->is_cuda()) {
-    torch::utils::cuda_lazy_init();
+  if (device) {
```

Comment:
should we move nullptr check into maybe_initialize_device?
then, the coding style is unified in all scenarios?